CCScriptWriter/CCScriptWriter.py

```diff
@@ -616,123 +560,30 @@ def outputToCoilSnakeProject(self):
                                 f = self.dataFiles[b]
                                 yamlData[e][s][n][a] = "{}.l_{}".format(f,
                                                                         hex(b))
-            csFile = open(os.path.join(o, fileName), "w")
             output = yaml.dump(yamlData, default_flow_style=False,
-                      Dumper=yaml.CSafeDumper)
-            output = re.sub("Event Flag: (\d+)",
-                   lambda i: "Event Flag: " + hex(int(i.group(0)[12:])), output)
-            csFile.write(output)
-            csFile.close()
+                    Dumper=yaml.CSafeDumper)
+            output = re.sub(r"Event Flag: (\d+)",
+                lambda i: "Event Flag: " + hex(int(i.group(0)[12:])), output)
+            with open(os.path.join(o, fileName), "w", encoding="utf8") as csFile:
+                csFile.write(output)
 
     # Gets the text at a specified location in memory; if stop is specified, it
     # will forcibly stop looking once it is reached. The third parameter
     # specifies whether the data block is a normal block, a coffee scene type
     # block or a staff list block.
-    def getText(self, i, stop=None, dataType=False):
+    def getText(self, i, stop=None, dataType=TextType.Normal):
```

Comment:
Seems like the linter didn't complain about turning *all* the doc comments into doc strings? Probably not a super big deal, but 🤷‍♀️